resources/scenic-analysis.wrattler

```diff
@@ -1,42 +1,60 @@
 # Testing the Broken Window hypothesis: are less scenic neighbourhoods linked to more crime?
+<br /> 
+The Broken Window hypothesis proposes that visible signs of crime and civil disorder creates an environment that encourages further crime and disorder.
+The hypothesis has been reponsible for promoting controversial policing policies throughout the 1990s. 
+Perhaps the most notable application of the theory was the adoption of the *stop-and-frisk* practise in New York City. 
+Whilst application of such policies have been linked to decline in crime rates, great skeptism surrounds the validity of such a relationship
+because of other changes taking place at that time, including the number of police officers, periods of economic growth and a reduction in poverty.
 
-## Crime data
+We wish to study the relationship between crime rates and scenicness in areas across London, as well as accounting for other indices of deprivation.
```

Comment:
Change to "We wish to study the relationship between crime rates and scenicness in areas across London, while controlling for other factors that could be linked with crimes, such as income and employment deprivation and population density."